tests/test_realm.py

```diff
@@ -26,6 +27,7 @@ async def test_realm_realmCreated(websocket, context_id, html):
 
     await send_JSON_command(
         websocket, {
+            "id": 21,
```

Comment:
You can omit the id and it will be assigned automatically

Reply:
you don't need it in this test, just FYI. `send_JSON_command` returns a command id, which can be used later on for assertions